extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/CdcSourceP.java

```diff
@@ -273,24 +371,40 @@ public OffsetStorageReader offsetStorageReader() {
         }
     }
 
-    private static long extractTimestamp(SourceRecord record) {
-        if (record.valueSchema().field("ts_ms") == null) {
-            return 0L;
-        } else {
-            return ((Struct) record.value()).getInt64("ts_ms");
-        }
+    private static String getName(Properties properties) {
+        return (String) properties.get(NAME_PROPERTY);
     }
 
     private static RetryStrategy getRetryStrategy(Properties properties) {
         RetryStrategy strategy = (RetryStrategy) properties.get(RECONNECT_BEHAVIOR_PROPERTY);
         return strategy == null ? DEFAULT_RECONNECT_BEHAVIOR : strategy;
     }
 
+    private static long getCommitPeriod(Properties properties) {
+        String periodString = (String) properties.get(COMMIT_PERIOD_MILLIS_PROPERTY);
+        long periodMillis = periodString == null ? DEFAULT_COMMIT_PERIOD_MS : Long.parseLong(periodString);
+        return TimeUnit.MILLISECONDS.toNanos(periodMillis);
+    }
+
     private static boolean getClearStateOnReconnect(Properties properties) {
         String s = (String) properties.get(RECONNECT_RESET_STATE_PROPERTY);
         return Boolean.parseBoolean(s);
     }
 
+    private static void log(Object value, String property, String name, ILogger logger) {
+        if (logger.isInfoEnabled()) {
+            logger.info(name + " has `" + property + "` to " + value);
+        }
+    }
+
+    private static long extractTimestamp(SourceRecord record) {
+        if (record.valueSchema().field("ts_ms") == null) {
+            return 0L;
```

Comment:
We must use `NO_NATIVE_TIME` in this case, as the `flatMapEvent`'s javadoc specifies.